### ISSUE TITLE
Some internal errors encountered during use

### ISSUE THREAD

**Issue by Sagners** [Pythagora-io/gpt-pilot#230]
Hi,everyone.I don't know if anyone has been in a similar situation to mine.In a nutshell, When I try to develop some complex software, the program always encounters some bugs that cannot be overcome, I checked the following, it seems that basically the relatively "static" parts of the project are not adaptable?
  Is there a way to solve these problems? Or, is there a way to do a version rollback that allows me to try something else to circumvent these issues instead of getting into the software I was working on last time and getting into a bug and exiting?
  After reading our documentation, maybe "step" can solve my problem, but I still go directly after using it, and then crash directly, and I can't fall back to a "safe location"

![image](https://github.com/Pythagora-io/gpt-pilot/assets/53614951/63ddcc9e-6646-4466-8943-e45600e9eec8)
![image](https://github.com/Pythagora-io/gpt-pilot/assets/53614951/3a4d0be8-65f0-48d3-899f-f91c2f1325bd)

**Comment by AndyLdl** [comment]
I have same exception

**Comment by Umpire2018** [comment]
Please try to add encoding in `pilot/helpers/files.py` to see if it's working.
![image](https://github.com/Pythagora-io/gpt-pilot/assets/138990495/8399cc94-80bb-4c24-9d6f-bc6d241daa8c)

**Comment by Sagners** [comment]
Thank you very much！
The problem with the encoding format had been successfully resolved。
But the problem of this error report is still not solved
<img width="692" alt="image" src="https://github.com/Pythagora-io/gpt-pilot/assets/53614951/5e3126ea-8dba-41ab-a1c3-4c25f12b8f3a">

![image](https://github.com/Pythagora-io/gpt-pilot/assets/53614951/1587b925-266a-40d0-8046-f99b6b8238bc)
I tried to reproduce this issue several times, and it seems that the reason it appears is because he will load the last error message, and then cause this problem (personal guess)
In summary, it seems that there will always be a no-brainer error before this happens, such as a request to reload OpenAPI

**Comment by Umpire2018** [comment]
@Sagners Thank you for your patience.

please modify `pilot/logger/logger.py` 
![image](https://github.com/Pythagora-io/gpt-pilot/assets/138990495/94ad3757-1ed6-4b45-a114-5d7cb41bec31)
change `logging.INFO` to logging.DEBUG` .

You will find detailed log in `pilot/logger/debug.log`.
Upload debug.log here and we will look into it.

**Comment by nalbion** [comment]
> Thank you very much！ The problem with the encoding format had been successfully resolved。 But the problem of this error report is still not solved <img alt="image" width="692" src="https://user-images.githubusercontent.com/53614951/276993901-5e3126ea-8dba-41ab-a1c3-4c25f12b8f3a.png">

That's odd - by the time it gets to `parse_agent_response` it should have already validated the response against the JSON schema

**Comment by sarahedwin442** [comment]
I want to use this golden medium to Appreciate Dr Kuda for making my dream come through. I was on the internet when I saw people posting and talking about Dr Kuda. On How he has help them in winning huge amounts in lottery ticket, My name is Sarah Edwin, this is my story on how I win $754.6 Million Dollars, After reading the article talking good about Dr Kuda I contacted Dr Kuda I told him to help me win the Powerball lottery, Then he respond to me by saying he will help me after working with Dr Kuda, and assured me that I will win the Powerball Lottery Draw, Which I believe. After working with Dr Kuda, he gave me a five digit number and ask me to go and play it which I did today, now i’m here shearing a testimony about this same man Dr Kuda Changed my life by helping me to win $754.6 Million Dollars all thanks to you Dr Kuda, I will keep talking about your good work, because you are a man of your word I hope people that really need this help will come across it for you to help them as well. If you want to win biggest in lottery contact Dr. Kuda today and be rest assured of winning lottery game, contact via email: drkudaspelltemple@gmail.com Or Call and Whatsapp Number +1 (346) 478-1991
